schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL,
+    species VARCHAR(50),
```

Comment:
- [ ] Kindly note that the table `animals` was already created in the previous activity. The objective of the question is to learn how to add a new column to an existing table using `ALTER TABLE`

Reply:
Thanks dear @LionRouge1 👏